website/versioned_docs/version-28.x/JestObjectAPI.md

```diff
@@ -837,42 +837,54 @@ This function is not available when using legacy fake timers implementation.
 
 ## Misc
 
-### `jest.setTimeout(timeout)`
```

Comment:
Only `jest.retryTimes()` entry was touched. `jest.setTimeout()` was simply moved under it.